From 21895145d25f3e1ffaac1d498995d37b04675455 Mon Sep 17 00:00:00 2001
From: Sami Shalayel <sami.shalayel@qt.io>
Date: Wed, 17 Sep 2025 15:20:56 +0200
Subject: [PATCH 057/239] CRA: review src/plugins/qmllint and src/plugins/qmlls

Mark all files as significant.

Pick-to: 6.9 6.8
Task-number: QTBUG-136187
Change-Id: I655241e8fd460f423c230b3ae2979246f4ffc841
Reviewed-by: Ulf Hermann <ulf.hermann@qt.io>
(cherry picked from commit 52e6456d6e71c3e93f7de7b3369738dc0d802b33)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/plugins/qmllint/qds/qdslintplugin.cpp                    | 1 +
 src/plugins/qmllint/qds/qdslintplugin.h                      | 1 +
 src/plugins/qmllint/quick/qquickliteralbindingcheck.cpp      | 1 +
 src/plugins/qmllint/quick/qquickliteralbindingcheck_p.h      | 1 +
 src/plugins/qmllint/quick/qquickvaluetypefromstringcheck.cpp | 1 +
 src/plugins/qmllint/quick/qquickvaluetypefromstringcheck_p.h | 1 +
 src/plugins/qmllint/quick/quicklintplugin.cpp                | 1 +
 src/plugins/qmllint/quick/quicklintplugin.h                  | 1 +
 src/plugins/qmlls/quick/qqmllsquickplugin.cpp                | 1 +
 src/plugins/qmlls/quick/qqmllsquickplugin_p.h                | 1 +
 10 files changed, 10 insertions(+)

diff --git a/src/plugins/qmllint/qds/qdslintplugin.cpp b/src/plugins/qmllint/qds/qdslintplugin.cpp
index 6fc87599c5..3d5d86d576 100644
--- a/src/plugins/qmllint/qds/qdslintplugin.cpp
+++ b/src/plugins/qmllint/qds/qdslintplugin.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant reason:default
 
 #include "qdslintplugin.h"
 
diff --git a/src/plugins/qmllint/qds/qdslintplugin.h b/src/plugins/qmllint/qds/qdslintplugin.h
index 6427f20b38..e1e7d8dec6 100644
--- a/src/plugins/qmllint/qds/qdslintplugin.h
+++ b/src/plugins/qmllint/qds/qdslintplugin.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant reason:default
 
 #ifndef QDSLINTPLUGIN_H
 #define QDSLINTPLUGIN_H
diff --git a/src/plugins/qmllint/quick/qquickliteralbindingcheck.cpp b/src/plugins/qmllint/quick/qquickliteralbindingcheck.cpp
index 47db2e33ca..ae6aaf4eb7 100644
--- a/src/plugins/qmllint/quick/qquickliteralbindingcheck.cpp
+++ b/src/plugins/qmllint/quick/qquickliteralbindingcheck.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant reason:default
 
 #include "qquickliteralbindingcheck_p.h"
 #include "qquickvaluetypefromstringcheck_p.h"
diff --git a/src/plugins/qmllint/quick/qquickliteralbindingcheck_p.h b/src/plugins/qmllint/quick/qquickliteralbindingcheck_p.h
index 31e735b6b8..02fc26c3d1 100644
--- a/src/plugins/qmllint/quick/qquickliteralbindingcheck_p.h
+++ b/src/plugins/qmllint/quick/qquickliteralbindingcheck_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKLITERALBINDINGCHECK_H
 #define QQUICKLITERALBINDINGCHECK_H
diff --git a/src/plugins/qmllint/quick/qquickvaluetypefromstringcheck.cpp b/src/plugins/qmllint/quick/qquickvaluetypefromstringcheck.cpp
index 40f1d674f9..96d8ab98da 100644
--- a/src/plugins/qmllint/quick/qquickvaluetypefromstringcheck.cpp
+++ b/src/plugins/qmllint/quick/qquickvaluetypefromstringcheck.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant reason:default
 
 #include "qquickvaluetypefromstringcheck_p.h"
 
diff --git a/src/plugins/qmllint/quick/qquickvaluetypefromstringcheck_p.h b/src/plugins/qmllint/quick/qquickvaluetypefromstringcheck_p.h
index 79575e80cf..f87fc25200 100644
--- a/src/plugins/qmllint/quick/qquickvaluetypefromstringcheck_p.h
+++ b/src/plugins/qmllint/quick/qquickvaluetypefromstringcheck_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant reason:default
 
 #ifndef QQUICKVALUETYPEFROMSTRINGCHECK_H
 #define QQUICKVALUETYPEFROMSTRINGCHECK_H
diff --git a/src/plugins/qmllint/quick/quicklintplugin.cpp b/src/plugins/qmllint/quick/quicklintplugin.cpp
index f06309917d..afa1a45292 100644
--- a/src/plugins/qmllint/quick/quicklintplugin.cpp
+++ b/src/plugins/qmllint/quick/quicklintplugin.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2022 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant reason:default
 
 #include "quicklintplugin.h"
 #include "qquickliteralbindingcheck_p.h"
diff --git a/src/plugins/qmllint/quick/quicklintplugin.h b/src/plugins/qmllint/quick/quicklintplugin.h
index 498d983e54..edf2cfbb5e 100644
--- a/src/plugins/qmllint/quick/quicklintplugin.h
+++ b/src/plugins/qmllint/quick/quicklintplugin.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2022 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant reason:default
 
 #ifndef QUICKLINTPLUGIN_H
 #define QUICKLINTPLUGIN_H
diff --git a/src/plugins/qmlls/quick/qqmllsquickplugin.cpp b/src/plugins/qmlls/quick/qqmllsquickplugin.cpp
index d311229df2..8ad904bf0a 100644
--- a/src/plugins/qmlls/quick/qqmllsquickplugin.cpp
+++ b/src/plugins/qmlls/quick/qqmllsquickplugin.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant reason:default
 
 #include "qqmllsquickplugin_p.h"
 #include <QtQmlLS/private/qqmllsutils_p.h>
diff --git a/src/plugins/qmlls/quick/qqmllsquickplugin_p.h b/src/plugins/qmlls/quick/qqmllsquickplugin_p.h
index da6bc6f827..69a83354fb 100644
--- a/src/plugins/qmlls/quick/qqmllsquickplugin_p.h
+++ b/src/plugins/qmlls/quick/qqmllsquickplugin_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant reason:default
 
 #ifndef QMLLSQUICKPLUGIN_H
 #define QMLLSQUICKPLUGIN_H
-- 
2.51.2

